src/lib/orionld/serviceRoutines/orionldPostEntity.cpp

```diff
@@ -186,9 +186,9 @@ bool kjNodeAttributeMerge(KjNode* sourceP, KjNode* updateP)
   // Go over the entire updateP tree and replace all those nodes in sourceP
   // Also, update the modDate node to the current date/time
   //
-  KjNode*  modDateP  = kjLookup(sourceP, "modDate");
-  KjNode*  mdP       = kjLookup(sourceP, "md");
-  KjNode*  mdNamesP  = kjLookup(sourceP, "mdNames");
+  KjNode*  modDateP  = kjLookup(sourceP, (char *) "modDate");
```

Comment:
Instead of all these typecasts to "char*", change kjLookup to have a "const char*" as second argument

Reply:
Okay

Reply:
I saw here, on the code, and already are const char* name